geodiff/src/sqlitedriver.h

```diff
@@ -0,0 +1,40 @@
+/*
+ GEODIFF - MIT License
+ Copyright (C) 2020 Martin Dobias
+*/
+
+#ifndef SQLITEDRIVER_H
+#define SQLITEDRIVER_H
+
+#include "geodiffutils.hpp"
+#include "geodiffdriver.h"
+
+/**
+ * Support for diffs between Sqlite-based files (including GeoPackage)
+ *
+ * Connection configuration:
+ *
+ * - for a single database use (not possible to call createChangeset())
+ *   - "base" = path to the database
+ *
+ * - for use with two databases (possible to call createChangeset())
+ *   - "base" = path to the 'base' database
+ *   - "modified" = path to the 'modified' database
```

Comment:
I am still not sure if the modified should not be part of createChangeset function and open will have just base?